ext/dom/node.c

```diff
@@ -693,14 +694,13 @@ int dom_node_prefix_write(dom_object *obj, zval *newval)
 					}
 					if (ns == NULL) {
 						ns = xmlNewNs(nsnode, nodep->ns->href, (xmlChar *)prefix);
+						if (UNEXPECTED(ns == NULL)) {
```

Comment:
Note to self: recheck if OOM is the only way in this path that this can return NULL.